sdk/demos/intel/fluid/shaders/shaderBitonicSort.comp

```diff
@@ -0,0 +1,47 @@
+#version 310 es
+precision mediump float;
+layout(local_size_x = 512, local_size_y = 1, local_size_z = 1) in;
+
+void GroupMemoryBarrierWithGroupSync()
+{
+    memoryBarrierShared();
+    barrier();
+}
+
+struct Grid
+{
+    int grid;
+    int padding1;
+    int padding2;
+    int padding3;
```

Comment:
The paddings are not necessary. The compiler will do it for you.

Reply:
Done.